doc/command_ref.rst

```diff
@@ -877,13 +874,13 @@ Makecache Command
 Mark Command
 -------------
 
-``dnf mark install <package-specs>...``
+``dnf mark install <package-spec>...``
```

Comment:
Here I suggest that we should accepts only package-name-specs. The code bellow requires change.

Reply:
Ok, but let's not add to this PR, it could also be considered a regression...